docs/index.md

```diff
@@ -196,18 +216,20 @@ Pulse uses these reports to calculate the metrics [Median time to recovery](metr
 
 Send the following information when reporting **incidents** to Pulse:
 
-| Field             | Description                                    | Format                                       |
-| ----------------- | ---------------------------------------------- | -------------------------------------------- |
-| identifier        | A unique identifier of the incident            | String                                       |
-| timestampCreated  | Time when the incident started or was detected | Number<br/>(Unix epoch timestamp in seconds) |
-| timestampResolved | Time when the incident was resolved            | Number<br/>(Unix epoch timestamp in seconds) |
+| Field             | Description                                      | Format                                       |
+| ----------------- | ------------------------------------------------ | -------------------------------------------- |
+| system            | Optional. System to assign to this report.       | String                                       |
```

Comment:
The `report` here seems very weird. Is not used anywhere else which will probably create confusion.
Would be nice to have same words in all the fields.

Reply:
I'm now using the verb "report" to mean the action of sending data about the events to Pulse. For example:

> [...] the recommended way of reporting the necessary events to Pulse is by using the Pulse CLI.

> Report an event to Pulse whenever your team deploys code to production [...]

But indeed, in this particular sentence "report" is used as a noun and can be mistaken with a new concept. I will rewrite to avoid this.